ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-servers.j2

```diff
@@ -18,6 +18,34 @@
 {%         if dhcp_server.dns_domain_name_ipv6 is arista.avd.defined %}
    dns domain name ipv6 {{ dhcp_server.dns_domain_name_ipv6 }}
 {%         endif %}
+{%         if dhcp_server.dns_server.ipv4_servers is arista.avd.defined %}
+{%             set dns_servers_cli = namespace(cli="dns server ipv4") %}
+{%             for server in dhcp_server.dns_server.ipv4_servers  %}
+{%                 set dns_servers_cli.cli = dns_servers_cli.cli ~ " " ~ server %}
+{%             endfor %}
+   {{ dns_servers_cli.cli }}
+{%         endif %}
+{%         if dhcp_server.dns_server.ipv6_servers is arista.avd.defined %}
+{%             set dns_servers_cli = namespace(cli="dns server ipv6") %}
+{%             for server in dhcp_server.dns_server.ipv6_servers  %}
+{%                 set dns_servers_cli.cli = dns_servers_cli.cli ~ " " ~ server %}
+{%             endfor %}
+   {{ dns_servers_cli.cli }}
+{%         endif %}
+{%         if dhcp_server.tftp_server.file.ipv4_file is arista.avd.defined %}
+{%             set tftp_file_cli = namespace(cli="tftp server file ipv4") %}
+{%             for file in dhcp_server.tftp_server.file.ipv4_file | arista.avd.natural_sort %}
+{%                 set tftp_file_cli.cli = tftp_file_cli.cli ~ " " ~ file %}
```

Comment:
Is this a valid format for a dhcp client?

Reply:
I get where you're coming from but I think that question is a bit too early.  This PR is just focused on EOS CLI.   All I can say for now is that this is valid EOS CLI and for the sake of this PR, I don't think we care if this is a valid format for a dhcp client.

Reply:
I think the issue for me it the use of " "  as a list separator.  The dhcp allows for a an opaque string with a length of 1-255 (len==u8). We are saying that the elements must be separated.  If a client needs a different separator, they are going to hack a value in list[0] and not use the others.  Can we specify a dist separator that defaults to " " or ","?